docs/cppintro/arrays.md

```diff
@@ -33,15 +33,15 @@ acestuia. În mod similar cu variabilele simple, ne trebuie un tip de date pe
 care acest tablou să-l stocheze, precum și dimensiunea pe care vrem să o
 atribuim acestui tablou.
 
-De exemplu, `#!cpp int v[101];` înseamnă ca am declarat un tablou cu $101$
```

Comment:
as lasa tot indexare de la 1, dar as pune comentariul pe care l-ai zis

Reply:
Indexarea de la 1 e mai ușoară, dar mai puțin corectă.

Reply:
cred ca pentru un incepator, e mult mai simplu sa inteleaga indexarea de la 1, putem eventual ca o solutie de compromis sa mentionam ambele, cu avantajele si dezavantajele lor. totusi, la toate concursurile, indexarea in probleme se da de la 1, chiar daca indexarea de la 0 e in teorie cea mai eficienta dpdv al memoriei. de asemenea, exista algoritmi in care e mult mai bine sa indexezi de la 1 (sume partiale de exemplu) si la fel algoritmi unde e mai bine sa indexezi de la 0 (de exemplu, lucrul cu biti). in 99% din cazuri in schimb, ambele sunt la fel de viabile